tests/rkt-monitor/main.go

```diff
@@ -56,8 +61,12 @@ func init() {
 	pidMap = make(map[int32]*process.Process)
 
 	cmdRktMonitor.Flags().BoolVarP(&flagVerbose, "verbose", "v", false, "Print current usage every second")
+	cmdRktMonitor.Flags().IntVarP(&repetitionNumber, "repetitions", "r", 1, "Numbers of benchmark repetitions")
```

Comment:
We have a similar usage doc in tests/rkt-monitor/README.md, and the two are now not in sync.